### ISSUE TITLE
CreateJobTest#shouldCreateJob is unstable

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#942]
The timestamp assertion of the test is a bit error prone. since in the test the timestamp is stored and return as result like this:
```java
        final Instant expectedTimestamp = Instant.now();

        brokerRule.onExecuteCommandRequest(ValueType.JOB, JobIntent.CREATE)
            .respondWith()
            .event()
            .key(123)
            .position(456)
            .timestamp(expectedTimestamp)
            .sourceRecordPosition(1L)
            .intent(JobIntent.CREATED)
            .value()
              .allOf((r) -> r.getCommand())
              .put("deadline", Protocol.INSTANT_NULL_VALUE)
              .put("worker", "")
              .done()
            .register();
```
The assertions looks like this:
```java
assertThat(metadata.getTimestamp()).isEqualTo(now);
```
Where `now` is a field which is also initialized via  `Instant.now();`

```java
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.191 s <<< FAILURE! - in io.zeebe.client.job.CreateJobTest
[ERROR] shouldCreateJob(io.zeebe.client.job.CreateJobTest)  Time elapsed: 0.043 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<...18-06-12T07:39:44.04[2]Z> but was:<...18-06-12T07:39:44.04[6]Z>
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at io.zeebe.client.job.CreateJobTest.shouldCreateJob(CreateJobTest.java:127)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)

```

**Comment by npepinpe** [comment]
Seems this was fixed by c2a15bd, closing (correct me if I'm wrong @saig0 )

**Comment by Zelldon** [comment]
Ok thanks then it seems that this a problem after rebasing my branch.
